.gitignore

```diff
@@ -8,11 +8,13 @@
 *.a
 *.cache
 *.dirstamp
+*.log
```

Comment:
These got dropped in the recent commit 293a928e62, but they were more specific:

    test-*.log
    test-*.trs

I think that is safer, just in case we ever get "precious" files (unlikely, but still..)

Reply:
I agree, and I actually prefer the more specific pattern.  I'll change it back.